src/packages/@app/components/App.svelte

```diff
@@ -101,9 +101,14 @@
       <Route path="/app/*" component={Dashboard} />
       <Route path="/*"><Navigate to="/app/" /></Route>
     </section>
+    <section slot="guestUser">
+      <Route path="/guest/*" component={Dashboard} />
+      <Route path="/*"><Navigate to="/guest" /></Route>
+    </section>
     <section slot="unauthorized">
       {#if 1}
         <Route path="/init" component={EntryPoint} />
+        <Route path="/init/*" component={Dashboard} />
         <Route path="/*"><Navigate to="/init" /></Route>
     {:else}
```

Comment:
if we will not make different routes then if let's say user has logged in as guest user and closes the app, and again comes so it should directly logs in as guest user in dashboard, and in absence of it, it will be directed to login flow.

Reply:
@gc-codes could you please see this. i think this can be confusing. or we can document "guestUser" vs "unauthorized"